api/resolvers/organizations.py

```diff
@@ -0,0 +1,53 @@
+from graphql import GraphQLError
+from sqlalchemy.orm import load_only
+
+from schemas.organizations import (
+	Organizations,
+	OrganizationsModel
+)
+
+from schemas.groups import (
+	Groups,
+	GroupsModel
+)
+
+
+# Resolvers
+def resolve_get_org_by_id(self, info, **kwargs):
+	"""Return an organization by its id"""
+	org_id = kwargs.get('id')
+	query = Organizations.get_query(info).filter(
+		OrganizationsModel.id == org_id
+	)
+	if not len(query.all()):
+		raise GraphQLError("Error, Invalid ID")
```

Comment:
Can all of the error messages go into functions in this file?  Easier to manage them all in one spot!

Reply:
It's a specific and short error message doesn't need to be put into a function, and it will never change